Everything/EverythingAPI.cs

```diff
@@ -123,13 +123,18 @@ public List<SearchResult> Search(string keyWord, CancellationToken token, SortOp
                 EverythingApiDllImport.Everything_SetSearchW(keyWord);
                 EverythingApiDllImport.Everything_SetOffset(offset);
                 EverythingApiDllImport.Everything_SetMax(maxCount);
+                
+                if(!EverythingApiDllImport.Everything_IsFastSort(sortOption))
+                {
+                    throw new InvalidOperationException("The Sort Option is not Fast Sort, it may take a long time to finish the query");
```

Comment:
should this be a prompt instead?

Reply:
It will be displayed like this
![image](https://user-images.githubusercontent.com/45326534/126752931-59d4a5fc-8c0a-4cea-81cc-c6d2b325093f.png)